bval-jsr/src/main/java/org/apache/bval/jsr/metadata/Meta.java

```diff
@@ -354,11 +354,12 @@ public boolean equals(Object obj) {
         if (!obj.getClass().equals(getClass())) {
```

Comment:
guess we would like to have a real equals/hashcode

what about just dropping both and use identity default impl, should be sufficient since meta should be cached per reflection instance in bval

Reply:
removed it